mountpoint-s3-client/tests/endpoint_config.rs

```diff
@@ -36,33 +35,126 @@ async fn run_test<F: FnOnce(&str) -> EndpointConfig>(f: F) {
     check_get_result(result, None, &body[..]).await;
 }
 
-#[test_case(AddressingStyle::Automatic)]
-#[test_case(AddressingStyle::Path)]
+#[test_case(AddressingStyle::Automatic, "test_default_addressing_style")]
+#[test_case(AddressingStyle::Path, "test_path_addressing_style")]
 #[tokio::test]
-async fn test_addressing_style_region(addressing_style: AddressingStyle) {
-    run_test(|region| EndpointConfig::new(region).addressing_style(addressing_style)).await;
+async fn test_addressing_style_region(addressing_style: AddressingStyle, prefix: &str) {
+    run_test(
+        |region| EndpointConfig::new(region).addressing_style(addressing_style),
+        &get_unique_test_prefix(prefix),
+        get_test_bucket(),
+    )
+    .await;
 }
 
 #[cfg(feature = "fips_tests")]
 #[tokio::test]
 async fn test_fips_mount_option() {
-    run_test(|region| EndpointConfig::new(region).use_fips(true)).await;
+    let prefix = get_unique_test_prefix("test_fips");
+    run_test(
+        |region| EndpointConfig::new(region).use_fips(true),
+        &prefix,
+        get_test_bucket(),
+    )
+    .await;
+}
+
+// Transfer acceleration do not work with path style
+#[cfg(feature = "tansfer_accelerate_tests")]
+#[tokio::test]
+async fn test_accelerate_mount_option() {
+    let prefix = get_unique_test_prefix("test_transfer_acceleration");
+    run_test(
+        |region| EndpointConfig::new(region).use_accelerate(true),
+        &prefix,
+        get_test_bucket(),
+    )
+    .await;
 }
 
 #[test_case(AddressingStyle::Automatic)]
 #[test_case(AddressingStyle::Path)]
 #[tokio::test]
 async fn test_addressing_style_dualstack_option(addressing_style: AddressingStyle) {
-    run_test(|region| {
-        EndpointConfig::new(region)
-            .addressing_style(addressing_style)
-            .use_dual_stack(true)
-    })
+    let prefix = get_unique_test_prefix("test_dual_stack");
```

Comment:
Maybe we could add the test prefix as a parameter here as well (similarly to `test_addressing_style_region()`)